src/gc/gcinterface.h

```diff
@@ -546,16 +546,16 @@ class IGCHeap {
     virtual HRESULT Initialize() = 0;
 
     // Returns whether nor this GC was promoted by the last GC.
-    virtual BOOL IsPromoted(Object* object) = 0;
+    virtual bool IsPromoted(Object* object) = 0;
 
     // Returns true if this pointer points into a GC heap, false otherwise.
-    virtual BOOL IsHeapPointer(void* object, BOOL small_heap_only = FALSE) = 0;
+    virtual bool IsHeapPointer(void* object, bool small_heap_only = false) = 0;
```

Comment:
I am pretty sure that C4800 is enabled. As I have said, the warning is not issues for all BOOL to bool conversions. It is issued only if the conversion results in extra native code to be produced.  

```
void bar(bool f)
{
}

void foo(BOOL f)
{
    bar(TRUE); // No warning
    bar(FALSE); // No warning
    bar(f); // WARNING!
}
```